### ISSUE TITLE
Typescript error if only 'extraReducers' is defined

### ISSUE THREAD

**Issue by trickeyd** [reduxjs/redux-toolkit#2058]
I have made a couple of slices that have no 'reducers' object, in favour of 'extraReducers'. This is in an effort to decrease the number actions fired throughout the app.

When I do this I get the following error:

` node_modules/@reduxjs/toolkit/dist/createSlice.d.ts:63:5
    63     reducers: ValidateSliceCaseReducers<State, CR>;
           ~~~~~~~~
    'reducers' is declared here.
`

I combat it by adding an empty reducers object.

Is this a deliberate decision?

Many thanks in advance.

**Comment by markerikson** [comment]
Yes, the `reducers` field is _always_ required for `createSlice` - if you don't need to define any actions here, you can leave it empty.

However, if there _aren't_ any actions being defined here, it actually doesn't make much sense to use `createSlice` at all, because its primary purpose _is_ to define additional actions.

In that case, you can just use `createReducer` instead.

**Comment by trickeyd** [comment]
Ah! createReducers - I totally forgot about this function. Yeah that makes a lot of sense. Thanks!

(Closing)
